nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -350,3 +435,464 @@ def test_widgets(self):
             assert 'state' in d
         assert 'version_major' in wdata
         assert 'version_minor' in wdata
+
+
+class TestRunCell(ExecuteTestBase):
+    """Contains test functions for ExecutePreprocessor.run_cell"""
+
+    @ExecuteTestBase.prepare_cell_mocks()
+    def test_idle_message(self, preprocessor, cell_mock, message_mock):
+        preprocessor.run_cell(cell_mock)
+        # Just the exit message should be fetched
+        message_mock.assert_called_once()
```

Comment:
This seems to be raising an error in 3.5 — any thoughts?

Reply:
That is weird... not sure why yet.

Reply:
Gah, since I changed to the built-in mock library for 3, this function isn't implemented in 3.5: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once

Reply:
I think it's good to switching to the built-in when possible — it sets us up for dropping the dependency when we drop support for python 2.